app/models/beard.rb

```diff
@@ -0,0 +1,35 @@
+class Beard < ActiveRecord::Base
+  belongs_to :user
+
+  STATUS = ['shaved', 'trimmed', 'unset']
+
+  validates :status,  presence: true
+  validates :user_id, presence: true
+
+  validate :valid_status
+
+  def update_status!(params)
+    self.status = params[:status] if params[:status]
+    self.save
+
+    self.toggle_massaged! if params[:is_massaged]
+  end
+
+  def toggle_massaged!
+    current_value = self.is_massaged
+    self.is_massaged = !!!current_value
```

Comment:
That's true. This is an overkill and I don't usually do it like this. I don't know why I added extra bangs anyway. XD

> On Apr 29, 2015, at 4:38 AM, MoreRon <notifications@github.com> wrote:
> 
> In app/models/beard.rb:
> 
> > +
> > +  validates :status,  presence: true
> > +  validates :user_id, presence: true
> > +
> > +  validate :valid_status
> > +
> > +  def update_status!(params)
> > +    self.status = params[:status] if params[:status]
> > +    self.save
> > +
> > +    self.toggle_massaged! if params[:is_massaged]
> > +  end
> > +
> > +  def toggle_massaged!
> > +    current_value = self.is_massaged
> > +    self.is_massaged = !!!current_value
> Overkill? One bang is enough to return true || false
> 
> —
> Reply to this email directly or view it on GitHub.
>